Makefile

```diff
@@ -14,6 +14,17 @@ install_deps:  ## Install all dependencies.
 setup:  ## Install all dependencies and setup pre-commit
 	make install_deps
 	pre-commit install
+	make .env
+
+.env:  ## Generate .env file
+	@echo "# AUTOGENERATED from .env.example please adjust" > $@
+	@echo "" >> $@
+	@cat .env.example >> $@
```

Comment:
Why don't you just copy .env.example as .env?
This means that the env example needs to be prefilled with mock values.

Reply:
I change the behavior so that .env.example is copied as .env